pages/components/Badge/badge.css

```
@@ -0,0 +1,27 @@
+.badge {
+    position: relative;
+    cursor: pointer;
+}
+
+.badge-count {
+    position: absolute;
+    background-color: #f44336;
+    color: #fff;
+    border-radius: 50%;
+    top: 0;
+    right: 0.4em;
+    font-size: 60%;
+    font-style: var(--font);
+    width: 25%;
+    height: 21%;
```

```suggestion
    width: 0.5rem;
    height: 0.4rem;
```
Avoid using % for width & heights